tests/lcs/core/chunks/delete.livecodescript

```diff
@@ -0,0 +1,38 @@
+script "CoreChunksDelete"
+/*
+Copyright (C) 2015 LiveCode Ltd.
```

Comment:
Argh! I made a specific effort to change that and somehow it didn't. SO ANNOYING.